metrics/sources/summary/summary.go

```diff
@@ -267,6 +267,8 @@ func (this *summaryMetricsSource) decodeEphemeralStorageStats(metrics *MetricSet
 		return
 	}
 	this.addIntMetric(metrics, &MetricEphemeralStorageUsage, storage.UsedBytes)
+	this.addIntMetric(metrics, &MetricNodeEphemeralStorageCapacity, storage.CapacityBytes)
```

Comment:
I don't see this metric in public docs: https://cloud.google.com/monitoring/api/metrics_kubernetes

Reply:
It's there: node/ephemeral_storage/total_bytes

case core.MetricNodeEphemeralStorageCapacity.MetricDescriptor.Name:
	point := sink.intPoint(timestamp, timestamp, value.IntValue)
	return createTimeSeries("k8s_node", nodeLabels, ephemeralstorageTotalBytesMD, point)

Reply:
No, this is ephemeral_storage/node_capacity: https://github.com/Stackdriver/heapster/blob/master/metrics/core/metrics.go#L662

Reply:
node/ephemeral_storage/total_bytes should be a total usage, not the capacity.

Reply:
The metric you point to is just an implementation detail. It's one of "core" metrics, which are an artifact from the times when Heapster supported multiple sinks, and each of them could specify their own metric naming. This metric is mapped to "node/ephemeral_storage/total_bytes" in Stackdriver sink (in a code snippet I included in the comment above).

"node/ephemeral_storage/total_bytes should be a total usage, not the capacity." - I don't really understand this comment. We have a separate metric for usage, total_bytes is a node capacity. We also used a capacity before this PR, I'm not changing it here.